source/includes/connection-snippets/scram-snippets/java-connection.java

```diff
@@ -0,0 +1,19 @@
+
+// begin scram connection
+import com.mongodb.ConnectionString;
+import com.mongodb.client.MongoClients;
+import com.mongodb.client.MongoClient;
+import com.mongodb.client.MongoDatabase;
+import com.mongodb.MongoClientSettings;
+
+// ...
+ConnectionString connString = new ConnectionString(
+    "mongodb+srv://<username>:<password>@<cluster-address>/test?w=majority"
+);
+MongoClientSettings settings = MongoClientSettings.builder()
+    .applyConnectionString(connString)
+    .retryWrites(true)
```

Comment:
Why do you not use the connection string for this, as with other drivers?  Did it not work?

Reply:
@ccho-mongodb might be able to shed some light on this, since this was his original code that I've adapted for this repo

Reply:
For the SCRAM connection snippets, these were taken verbatim from the drivers docs. For instance: https://docs.mongodb.com/ecosystem/drivers/java/#connect-to-mongodb-atlas

Reply:
Lets just use the connection string as a string and pass it to `MongoClients.create()` it will greatly simplify the creation of the client.

Reply:
Thanks for the suggestion - I've updated the snippet